unity-client/Assets/Scripts/MainScripts/DCL/Controllers/InputController/InputController.cs

```diff
@@ -222,7 +229,7 @@ public void Update_Trigger(InputAction_Trigger[] triggerTimeActions)
                     InputProcessor.FromKey(action, KeyCode.Alpha7, modifiers: InputProcessor.Modifier.FocusNotInInput);
                     break;
                 case DCLAction_Trigger.BuildEditModeChange:
-                    InputProcessor.FromKey(action, KeyCode.L, modifiers: InputProcessor.Modifier.FocusNotInInput);
+                    InputProcessor.FromKey(action, KeyCode.KeypadPlus, modifiers: InputProcessor.Modifier.FocusNotInInput);
```

Comment:
This won't work.
It will depend on the keyboard's layout.
We should avoid using any special character in here or upgrade to the new Unity's Input System

Reply:
Ups, I didn't know it, sorry! I changed the key associated (previously was the "L" key) because it was entering in conflict with the friends feature that used the same key. I have just change it to "K" key (it seems this key is not currently being used for nothing). Anyway this key is something temporal only for our testings, I guess in the future the BiW editor will be activated only by the panel you're acurrently implementing.

Reply:
thx ❤️ I can test it now :)